mlir-sycl/test/Conversion/SYCLToLLVM/sycl-methods-flat-to-llvm.mlir

```diff
@@ -0,0 +1,163 @@
+// RUN: sycl-mlir-opt -split-input-file -convert-sycl-to-llvm %s | FileCheck %s
+
+//===-------------------------------------------------------------------------------------------------===//
```

Comment:
NIT: Exceeds column width. 

Applies to most of these lines in this file.

Reply:
We don't have a strict column width limit for lit tests. I'd argue it's more readable this way. Do you have issues visualizing this in your Emacs?

Reply:
It wraps in my editor, which looks a bit weird. LLVM coding guidelines also suggest column width of 80. I get that limiting the column width of `CHECK` lines does not make much sense, but for these comment lines, we could stick with the limit, at least the newly added ones.